### ISSUE TITLE
Add support for more colour systems

### ISSUE THREAD

**Issue by drtrigon** [AbdealiJK/pycolorname#32]
Add support for more colour systems:

1. [CIE](https://en.wikipedia.org/wiki/CIE_1931_color_space#CIE_xy_chromaticity_diagram_and_the_CIE_xyY_colour_space) does not have names and uses the [Munsell color system](https://en.wikipedia.org/wiki/Munsell_color_system) in the given case, but also others, see e.g. [hyperphysics](http://hyperphysics.phy-astr.gsu.edu/hbase/vision/cie.html)
2. [NCS](https://en.wikipedia.org/wiki/Natural_Color_System) uses similarly to the Munsell color system a [color circle](https://de.wikipedia.org/wiki/Datei:NCS_Colour_Circle.tif) (the whole space is a sphere) to allow to name the colours
3. A very easy and popular system is given by [coloured pencils](https://de.wikipedia.org/wiki/Datei:Buntstifte_01_KMJ.jpg), see also [Wikipedia:WikiProject Color](https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Color)
4. The [Munsell color system](https://en.wikipedia.org/wiki/Munsell_color_system) allows to name colours by their position on the circle

Table with some Pantone NCS conversions: http://www.pintura.es/noticia/20/tabla/de/conversion/de/pantone/a/ncs/natural/color/systemen

Table with RGB Munsell conversations: http://www.munsellcolourscienceforpainters.com/ConversionsBetweenMunsellAndsRGBsystems.pdf

To summarize: Number 1. uses the Munsell color system for naming. For 2. no mathematical definition or way to convert it to another colour space - namely RGB - is available since its based on perception. Number 3. I did not find any nice RGB value table and probably never will. The last, number 4. looks promising we have a very detailed table given by Paul Centore from 2013 and is accepted by experts.

I would like to request to implement the Munsell color system according to the table given by Paul Centore which will allow us to assign the closest Munsell color to any given RGB value and using the Munsell color space description we can then name the color.

General remarks: NCS has 40, Munsell 20 colours to name along the circle. Humans can distinguish around 40000 different colours.

**Comment by drtrigon** [comment]
Addendum: There is also Color Index, see http://www.colour-index.com/technical-info#6 (cannot find a conversion to RGB, though)
